### PR TITLE
Milestones : Accessibility Report

### PR DESCRIPTION
**Pull request by AngomRanjan**
## Profile - Milestones : Accessibility Report

- Page titles - no issues found
- Image text alternatives - no issues found
- Text headings - [One alert for skip Heading](https://github.com/AngomRanjan/Profile/blob/4d09c848cc6b84cae5ea013aea2049417a7d9a7c/index.html#L34-L46)
- Color contrast - no issues found
- Resize - no issues found
- Interaction - no issues found
- Moving content - No issues found
- Multimedia - No issues found. The site doesn't contain any audio/ video file.
- The basic structure of the page - no issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   </head>
   <body>
     <header class="flex" id="header">
-      <div class="logo">
+      <div class="logo" id="logo">
```
Comment (sajid-munawar) [inline index.html:15]: - I appreciate your effort. It's working fine.

File: index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
         <a class="logo-text" href="#logo">Angom</a>
```
Comment (nati2323) [inline index.html:16]: - Great job the logo part is well done.

**Review by nati2323 — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
   </head>
```
Comment (arslanbisharat) [inline index.html:12]: - [ ] [OPTIONAL] The title is good but I would recommend you to use a brief title such as `Wek Developer at Jabscan ` or `Mark's portfolio`.

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (AngomRanjan) [reply index.html:12]: noted

**Pull request merged**